pkgs/applications/graphics/mandelbulber2/default.nix

```
@@ -0,0 +1,53 @@
+{ lib, mkDerivation, fetchFromGitHub, libpng, gsl, libsndfile, lzo,
+qmake, qttools, qtbase, qtmultimedia,
+opencl ? true, opencl-clhpp ? null, ocl-icd ? null }:
+
+assert opencl -> opencl-clhpp != null;
+assert opencl -> ocl-icd != null;
+
+mkDerivation rec {
+  pname = "mandelbulber2";
+  version = "2.19";
+
+  src = fetchFromGitHub {
+    owner = "buddhi1980";
+    repo = "mandelbulber2";
+    rev = version;
+    sha256 = "10kq30mxryq78kgsrqwpmws4knz7yfzhdq4zhl28rrz2k1bydgij";
+  };
+
+  nativeBuildInputs = [ qmake qttools ];
+  buildInputs = [ qtbase qtmultimedia libpng gsl libsndfile lzo ] ++
+    lib.optionals opencl [ opencl-clhpp ocl-icd ];
+
+  sourceRoot = "source/mandelbulber2";
+
+  qmakeFlags = [ "DEFINES+=SHARED_DIR=\\\\\\\"\${out}/share/mandelbulber2\\\\\\\""
```

You can use `''` strings to get rid of slash in each of `\"` and `\\`. You can also use `${placeholder "out"}` to avoid another one.
```suggestion
  qmakeFlags = [ ''DEFINES+=SHARED_DIR=\\\"${placeholder "out"}/share/mandelbulber2\\\"''
```

---

Won’t `SHARED_PATH`, as described in https://github.com/buddhi1980/mandelbulber2/blob/ce5f5d789c78ba76ea36d0efeb79f2f2b3ca643d/mandelbulber2/deploy/README#L137 work?

```suggestion
  qmakeFlags = [ "SHARED_PATH=${placeholder ''out''}" ];
```

https://github.com/buddhi1980/mandelbulber2/blob/ce5f5d789c78ba76ea36d0efeb79f2f2b3ca643d/mandelbulber2/qmake/installer.pri#L7-L9
https://github.com/buddhi1980/mandelbulber2/blob/ce5f5d789c78ba76ea36d0efeb79f2f2b3ca643d/mandelbulber2/src/system.hpp#L48-L58